### DISCUSSION TITLE
Possible defect: owner of original repo was able to push to branch/PR I opened from fork of code

### DISCUSSION THREAD

**Discussion by Trevoke** [community/community#37546] [Pull Requests] [answered]
This branch has two commits, one I made, from which I opened the PR, and one the owner of the original repo made.

I did not give that person permissions to add code to my repository so I'm very confused as to how this happened. Is this ... Intentional ?

https://github.com/WeiQiPro/IsometricGoban/compare/master...Trevoke:IsometricGoban:separation-of-concerns

**Comment by DenverCoder1** [comment, marked as answer]
There is a checkbox when you open a PR for "Allow edits from maintainers". You probably had a pull request with that option enabled.

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork

![image](https://user-images.githubusercontent.com/20955511/198390220-0392a9b5-9e75-4122-8cc6-f255ae906b74.png)
